### DISCUSSION TITLE
clang: error: unsupported option '-fopenmp' on bundle install

### DISCUSSION THREAD

**Discussion by Overload119** [libvips/ruby-vips#298] [Q&A]
**Questions or suggestions**

We are using [Discussions](https://github.com/libvips/ruby-vips/discussions) for questions and suggestions for new features. Issues are only for bug reports.

**Describe the bug**

Unable to gem install.

`clang: error: unsupported option '-fopenmp'`

**To Reproduce**

bundle

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
Mac OSX

**Additional context**

**Comment by sled** [comment]
The apple provided clang compiler does not support this flag, if you use a newer version it should work:

```
brew install llvm libomp
```

**Comment by jcupitt** [comment]
This sounds like a long-standing issue with imagemagick. It adds `-fopenmp` to the compile line of anything which links to it, but `-fopenmp` is a gcc flag, so you can only build downstream projects with gcc.

Though I'm unsure why gem would be trying to build something that links to magick, you'd need to provide more information.

As @sled says, a newer clang would help too.
